packages/material-ui/src/Popover/Popover.test.js

```diff
@@ -435,16 +431,12 @@ describe('<Popover />', () => {
     });
 
     it("should use anchorEl's parent body as container if container not provided", async () => {
-      openPopover(undefined, true).then(() => {
-        assert.strictEqual(
-          wrapper
-            .dive()
-            .find(Modal)
-            .props().container,
-          window.document.body,
-          "should use anchorEl's parent body as Modal container",
-        );
-      });
+      await openPopover(undefined, true);
```

Comment:
```suggestion
      await openPopover(undefined);
```
second argument was removed

Reply:
Thanks